config/manager/manager.yaml

```diff
@@ -72,4 +72,5 @@ spec:
           value: STATUS_ADDDON_IMAGE_NAME_PLACEHOLDER
         args:
         - "controller"
+        - "--agent-v=2"
```

Comment:
BTW, I expect this to get overwritten soon as we plumb control over logging through higher levels.